Jellyfin.Api/Controllers/LibraryController.cs

```diff
@@ -335,17 +333,11 @@ public async Task<ActionResult> RefreshLibrary()
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     public ActionResult DeleteItem(Guid itemId)
     {
-        var isApiKey = User.GetIsApiKey();
```

Comment:
Is this properly respected in you changes? If an API key is used Items should always be returned